Geometry/TrackerCommonData/data/PhaseI/pixfwd.xml

```diff
@@ -20,6 +20,10 @@
 
  <Constant name="ZCylinder"      value="[AnchorZ]"/>
 
+ <Constant name="FlexCableDiskRmin" value="117.35*mm"/>
+ <Constant name="FlexCableDiskRmax" value="167.9*mm"/>
+ <Constant name="FlexCableDiskThickness" value="0.3*mm"/> <!-- This is actually *half* thickness -->
```

Comment:
@Michael-Krohn @krose13 see the code comment that I added here. This might have to be halved, since the material used has already has a fairly high density and not all of the cable disk is actually covered by cable.

We have the question here whether we should include the mass of cables *outside* this volume in this volume (to get the total mass correct) or not. But here I think we should not, since we might move material from less critical regions right into the active area.